conf/messages.en

```diff
@@ -328,7 +328,8 @@ platformOperatorUpdated.link2.text = Return to manage your digital platform repo
 platformOperatorAdded.title = Platform operator added
 platformOperatorAdded.panel.heading = {0} added
 platformOperatorAdded.panel.p1 = The platform operator ID is {0}
-platformOperatorAdded.p1 = {0} has been successfully added as a platform operator. We have sent a confirmation email to {1} and {2}.
+platformOperatorAdded.p1a = {0} has been successfully added as a platform operator. We have sent a confirmation email to {1} and {2}.
```

Comment:
I would name this differently to be more clear, e.g. platformOperatorAdded.p1.two.emails platformOperatorAdded.p1.single.email
Hopefully you can come up with better names.

Reply:
Done.

Reply:
Done.